python/ray/tune/examples/pbt_transformers/pbt_transformers.py

```diff
@@ -102,17 +102,17 @@ def get_model():
     )
 
     tune_config = {
-        "per_device_train_batch_size": 32,
+        "per_device_train_batch_size": tune.choice([16, 32, 64]),
         "per_device_eval_batch_size": 32,
-        "num_train_epochs": tune.choice([2, 3, 4, 5]),
-        "max_steps": 1 if smoke_test else -1,  # Used for smoke test.
+        "num_train_epochs": 2 if smoke_test else tune.choice([2, 3, 4, 5]),
     }
 
     scheduler = PopulationBasedTraining(
         time_attr="training_iteration",
         metric="eval_acc",
         mode="max",
         perturbation_interval=1,
+        synch=True,
```

Comment:
We could also pass this as a **kwargs parameter so that the regular example has sync=False - happy either way